src/UniversalRewardsDistributor.sol

```diff
@@ -166,18 +170,6 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
         emit EventsLib.RootUpdaterSet(updater, active);
     }
 
-    /// @notice Revokes the pending root of a given distribution.
-    /// @dev This function can only be called by the owner of the distribution at any time.
-    /// @dev Can be frontrunned by triggering the `acceptRoot` function in case the timelock has passed. This if the
-    /// `owner` responsibility to trigger this function before the end of the timelock.
-    function revokeRoot() external onlyOwner {
-        require(pendingRoot.submittedAt != 0, ErrorsLib.NO_PENDING_ROOT);
-
-        delete pendingRoot;
-
-        emit EventsLib.RootRevoked();
-    }
-
```

Comment:
Is it expected? I don't think it corresponds to what we agreed upon lol

Reply:
Ive moved the function, github issue here

Reply:
To group all updaters functions together